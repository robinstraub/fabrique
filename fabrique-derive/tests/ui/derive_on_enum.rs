use fabrique_derive::Factory;

#[derive(Factory)]
enum Anvil {
    Heavy,
    Light,
}

fn main() {}