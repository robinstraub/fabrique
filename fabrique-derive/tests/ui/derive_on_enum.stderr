error: Factory can only be derived from named structs, enum given
 --> tests/ui/derive_on_enum.rs:4:1
  |
4 | / enum Anvil {
5 | |     Heavy,
6 | |     Light,
7 | | }
  | |_^
